lib/beanstalkd_view/beanstalkd_utils.rb

```diff
@@ -34,6 +34,13 @@ def job_to_hash(job)
       ret_value['body'] = job.body.inspect
       ret_value
     end
+
+    def tubes_tree(names)
```

Comment:
[question] Where is this used?

I'd prefer not to extend any core types, could there just be a simpler merge utility function?